client/src/login/Login.js

```diff
@@ -126,6 +126,25 @@ export default class Login extends Component {
                     </ButtonRow>
                 </Form>
             </div>
-        );
+          );
+        } else {
+          if (mailtrainConfig.isAuthenticated) {
+            return (
```

Comment:
Why is there a logout on the login page? (And only for CAS?)

Reply:
Ready https://github.com/Mailtrain-org/mailtrain/pull/960/commits/99e3f12c24e6392d9646e4141df03d382d74ca61